<!DOCTYPE html>
<html lang="en">

<!-- Head tag -->
<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!--Description-->
    
        <meta name="description" content="Team Members

li { font:22px &#39;Open Sans&#39;;margin-top:10px; }
div.img
  {
  margin:40px 50px;
  border:1px solid #bebebe;
  height:auto;
  width:auto;
 ">
    

    <!--Author-->
    
        <meta name="author" content="Wenjing Huang">
    

    <!--Open Graph Title-->
    
        <meta property="og:title" content="Season"/>
    

    <!--Open Graph Description-->
    

    <!--Open Graph Site Name-->
    <meta property="og:site_name" content="Season"/>

    <!--Type page-->
    
        <meta property="og:type" content="website" />
    

    <!--Page Cover-->
    

    <meta name="twitter:card" content="summary" />
    

    <!-- Title -->
    
    <title>Background&Motivation</title>

    <!-- Bootstrap Core CSS -->
    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.2/css/bootstrap.min.css" integrity="sha384-y3tfxAZXuh4HwSYylfB+J125MxIs6mR5FOHamPBG064zB+AFeWH94NdvaCBm8qnd" crossorigin="anonymous">

    <!-- Custom Fonts -->
    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" type="text/css">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="//oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="//oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

    <!-- Gallery -->
    <link href="//cdnjs.cloudflare.com/ajax/libs/featherlight/1.3.5/featherlight.min.css" type="text/css" rel="stylesheet" />

    <!-- Custom CSS -->
    <link rel="stylesheet" href="../css/style.css">

    <!-- Google Analytics -->
    


</head>


<body>

<div class="bg-gradient"></div>
<div class="bg-pattern"></div>

<!-- Menu -->
<!--Menu Links and Overlay-->
<div class="menu-bg">
    <div class="menu-container">
        <ul>
            
            <li class="menu-item">
                <a href="../">
                    Home
                </a>
            </li>
            
            <li class="menu-item">
                <a href="../Background">
                    Background&amp;Motivation
                </a>
            </li>
            
            <li class="menu-item">
                <a href="../Design">
                    Design
                </a>
            </li>
            
            <li class="menu-item">
                <a href="../Simulation">
                    Simulation
                </a>
            </li>
            
            <li class="menu-item">
                <a href="../Material">
                    Materials&amp;Methods
                </a>
            </li>
            
            <li class="menu-item">
                <a href="../Results">
                    Results&amp;Discussion
                </a>
            </li>
            
            <li class="menu-item">
                <a href="../Future">
                    Perspective
                </a>
            </li>
            
            <li class="menu-item">
                <a href="../Team">
                    Team
                </a>
            </li>
            
        </ul>
    </div>
</div>

<!--Hamburger Icon-->
<nav>
    <a href="#menu"> MENU</a>
</nav>

<div class="container">

    <!-- Main Content -->
    <div class="row">
    <div class="col-sm-12">

        <!--Title and Logo-->
        <header>
    <div class="logo">
        <a href="../"><img src="logo.png"></a>
        
    </div>
</header>

        <section class="main">
            
<div class="post">

    <div class="post-header">
        <h1 class="title">
            <a href="../Background/index.html">
                BACKGROUND&MOTIVATION
            </a>
        </h1>
        <div class="post-info">

            
        </div>
    </div>

    <div class="content">

        <!-- Gallery -->
        

        <!-- Post Content -->
        
<!doctype html>
<html>
<head>
<link rel="shortcut icon" href="my.ico">
<title>Background</title>
<style>
}
div.img
  {
  margin:20px 20px;
  height:auto;
  width:auto;
  float:middle;
  text-align:center;
  }
  div.img img
  {
  display:inline;
  margin:0px;
  border:10px solid #bebebe;
  vertical-align:middle
  }
div.img a:hover img
  {
  border:1px solid #333333;
div.img desc
  {
  float:middle;
  font-weight:weight;
  width:100px;
  font-size:30px;
  margin:10px;
  font-family: "Orbitron", sans-serif;
  cursor: pointer;
  text-align:center;
  }
</style>
</head>
<body>
<h1>1.	What is Z-DNA?</h1>
<p>Z-DNA is one of the many possible double helical structures of DNA. It is a left-handed double helical structure in which the double helix winds to the left in a zig-zag pattern (instead of to the right, like the more common B-DNA form). Z-DNA is thought to be one of three biologically active double helical structures along with A- and B-DNA.</p>
<h2>Structure</h2>
<p>Z-DNA is quite different from the right-handed forms. In fact, Z-DNA is often compared against B-DNA in order to illustrate the major differences. The Z-DNA helix is left-handed and has a structure that repeats every 2 base pairs .(Fig.1) The major and minor grooves, unlike A- and B-DNA, show little difference in width. Formation of this structure is generally unfavorable, although certain conditions can promote it; such as alternating purine-pyrimidine sequence (especially poly(dGC)2), negative DNA supercoiling or high salt and some cations (all at physiological temperature, 37 Â°C, and pH 7.3-7.4). Z-DNA can form a junction with B-DNA (called a "B-to-Z junction box") (Fig.2) in a structure which involves the extrusion of a base pair.<sup>[3]</sup> The Z-DNA conformation has been difficult to study because it does not exist as a stable feature of the double helix. Instead, it is a transient structure that is occasionally induced by biological activity and then quickly disappears.<sup>[4]</sup><p>

<br><br>
<div class="img">
  <a target="_blank" href="figure1.jpg" class="img">
  <img src="figure1.jpg" alt="Ballade" width="400" height="360">
  </a>
    <div class="desc">
	  Fig. 1 Structure of the Z-DNA sequence d(CGCGCG) <sup>[1]</sup>(Wang et al. 1979) 
	</div>
  </div>
 <div style="clear:both"></div>

<br><br>
<div class="img">
  <a target="_blank" href="figure2.jpg" class="img">
  <img src="figure2.jpg" alt="Ballade" width="360" height="420">
  </a>
     <div class="desc">
	  Fig.2 Structure of an oligonucleotide having a B-Z junction <sup>[2]</sup>(Ha et al., 2005) 
	</div>
 
  </div>
 <div style="clear:both"></div>

<h2>Predicting Z-DNA structure</h2>
<p>It is possible to predict the likelihood of a DNA sequence forming a Z-DNA structure. An algorithm for predicting the propensity of DNA to flip from the B-form to the Z-form, ZHunt, was written by Dr. P. Shing Ho in 1984 (at MIT).<sup>[5]</sup> This algorithm was later developed by Tracy Camp, P. Christoph Champ, Sandor Maurice, and Jeffrey M. Vargason for genome-wide mapping of Z-DNA (with P. Shing Ho as the principal investigator).<sup>[6]</sup></p>


<h1>2.	Biological significance of Z-DNA</h1>
<p>Though Z-DNA was firstly discovered in vitro, recently many evidences revealed that Z-DNA existed and may possessed diversity of functions in vivo including gene expression and regulation, chromosomal breaks, recombination, antivirus defense and virus generations and so on.</p>
<p>Z-DNA is commonly believed to provide torsional strain relief (supercoiling) while DNA transcription occurs.<sup>[7]</sup> The potential to form a Z-DNA structure also correlates with regions of active transcription. A comparison of regions with a high sequence-dependent, predicted propensity to form Z-DNA in human chromosome 22 with a selected set of known gene transcription sites suggests there is a correlation.<sup>[8]</sup>
</p>

<h1>3.	DNA Topology</h1>
<p>A typical DNA molecule consists of two complementary polynucleotide chains that are multiply interwound, forming a double helix. Topological aspects of DNA structure arise primarily from the fact that the two DNA strands are repeatedly intertwined. </p>

<p>The fundamental topological parameter of a covalently closed circular DNA is called the linking number (Lk). Assume that one DNA strand is the edge of an imaginary surface and count the number of times that the other DNA strand crosses this surface. The algebraic sum of all intersections is the Lk.<sup>[9]</sup>  Associated with linking number, we were curious what will happen if we mix two complementary mini circles because of the topological constraint should prohibit them to form only B-form conformation. </p>
<h2>DNA catenane</h2>
<p>Catenane is a mechanically-interlocked molecular architecture consisting of two or more interlocked macro-cycles. The unique mechanic bonds provide catenanes with special traits when it comes to explore their properties. The approaches of synthesizing them also yield new ideas to chemists.
DNA has been commonly known as the molecule unit to assemble catenanes, called DNA catenanes, since decades ago. It was first learned as a common cellular feature. <sup>[10]</sup> It is well-known that catenanes are the intermediates in the terminal stage of replication of circular DNA<sup>[11]</sup>, such as plasmid. Perhaps the most striking example of catenation is found in kinetoplasts where DNA is a network of thousands of linked rings. <sup>[12]</sup> DNA catenanes in nature were of large molecular weight and complex topological structure, which endowed controllable and valuable functions in biological activities.
With the rapid development of DNA nanotechnology, artificial fabrication of DNA catenanes has attracted more and more attention in this field. Compared with natural DNA catenanes, artificial DNA catenanes are smaller and topologically simpler. Generally single-stranded (ss) DNA, whose length is always less than 150 nt, was used to fabricate the nanostructures. Using elaborate design, people could make two or three ring catenanes with ssDNA which are controllable to participate in various activities on nano-scale, such as molecular motors. It is reported an efficient approach for preparing linear three-ring catenanes (L3C) composed of single-stranded DNA. <sup>[13]</sup> Examples like catenane rotary motor with controlled directionality <sup>[14]</sup> and oscillator controlled by pH <sup>[15]</sup> have also been reported in the past few years. Meanwhile, studies on the topological state of artificial DNA catenanes e.g. secondary structure <sup>[16]</sup> also provide new insight into DNA basic research.
</p>


<br><br>
<div class="img">
  <a target="_blank" href="figure3.jpg" class="img">
  <img src="figure3.jpg" alt="Ballade" width="450" height="360">     
  </a>
  <div class="desc">
  Fig. 3 Mathematical biological model of DNA catenane<sup>[17]</sup> 
	</div>
  </div>
 <div style="clear:both"></div>

<p>Our team had done DNA catenanes which successfully gained the DNA Triad(three-rings DNA catenanes )and the DNA Olympic rings by previous team members in 2014. Itâs worth continuing to go deep into DNA catenanes for the further use pragmatically.</p>


<h1>4.	Our purpose</h1>

<p>Currently, most of approaches to Z-DNA synthesis using chemical modification which has negative effect on the purity and functions. Meanwhile, most methods were with low yield. Inspired by the structure of DNA rings, we are going to building a DNA ring with the structure of Z-DNA which provides better stability and purity. Whatâs more, we hope to extend this method of Z-DNA synthesis to RNA for its functions in genetic transcription. We firmly believed that our ideas would infuse new blood to nucleic acid technology and provide scientists with inspiration to develop nucleic acid technology. Specified purposes are listed as follow:</p>

<p>1.Prepare B-Z chimeras and obtain the minimum length to form B-Z chimera, which is the main object of our project.</p>  
<p>2.Research whether B-D chimera will promote the transcription.</p>  
<p>3.Extend this method of Z-DNA synthesis to RNA and build Z-B RNA as our development goals.</p>  


<h1>References</h1>
<ol>	
<li style="word-break:break-all">Stephen Neidle. Principles of Nucleic Acid Structure.</li>  
<li style="word-break:break-all">Stephen Neidle. Principles of Nucleic Acid Structure.</li>  
<li style="word-break:break-all">de Rosa M, de Sanctis D, Rosario AL, Archer M, Rich A, Athanasiadis A, Carrondo MA (2010-05-18). "Crystal structure of a junction between two Z-DNA helices". Proc Natl Acad Sci USA. 107 (20): 9088â9092.</li>  
<li style="word-break:break-all">Zhang H, Yu H, Ren J, Qu X (2006). "Reversible B/Z-DNA transition under the low salt condition and non-B-form polydApolydT selectivity by a cubane-like europium-L-aspartic acid complex". Biophysical Journal. 90 (9): 3203â3207.</li>  
<li style="word-break:break-all">Ho PS, Ellison MJ, Quigley GJ, Rich A (1986). "A computer aided thermodynamic approach for predicting the formation of Z-DNA in naturally occurring sequences". EMBO Journal. 5 (10): 2737â2744.</li>  
<li style="word-break:break-all">Champ PC, Maurice S, Vargason JM, Camp T, Ho PS (2004). "Distributions of Z-DNA and nuclear factor I in human chromosome 22: a model for coupled transcriptional regulation".Nucleic Acids Res. 32 (22): 6501â6510.</li>  
<li style="word-break:break-all">Ha SC, Lowenhaupt K, Rich A, Kim YG, Kim KK (2005). "Crystal structure of a junction between B-DNA and Z-DNA reveals two extruded bases". Nature. 437 (7062): 1183â1186. </li>  
<li style="word-break:break-all">Champ PC, Maurice S, Vargason JM, Camp T, Ho PS (2004)."Distributions of Z-DNA and nuclear factor I in human chromosome 22: a model for coupled transcriptional regulation". Nucleic Acids Res. 32(22): 6501â6510. </li>  
<li style="word-break:break-all">Mirkin, Sergei M. "DNA topology: fundamentals." eLS (2001).</li>
<li style="word-break:break-all">Vologodskii, A. V.; Cozzarelli, N. R., Monte Carlo analysis of the conformation of DNA catenanes. J Mol Biol 1993, 232 (4), 1130-40.</li>
<li style="word-break:break-all">Sundin, O.; Varshavsky, A., Arrest of segregation leads to accumulation of highly intertwined catenated dimers: dissection of the final stages of SV40 DNA replication. Cell 1981, 25 (3), 659-69.</li>
<li style="word-break:break-all">Sturm, N. R.; Simpson, L., Partially edited mRNAs for cytochrome b and subunit III of cytochrome oxidase from Leishmania tarentolae mitochondria: RNA editing intermediates. Cell 1990, 61 (5), 871-8.</li>
<li style="word-break:break-all">Li Q, Wu G, Wu W, et al. Efficient Synthesis of Topologically Linked ThreeâRing DNA Catenanes[J]. ChemBioChem, 2016.</li>
<li style="word-break:break-all">Lu, C. H.; Cecconello, A.; Elbaz, J.; Credi, A.; Willner, I., A three-station DNA catenane rotary motor with controlled directionality. Nano letters 2013, 13 (5), 2303-8.</li>
<li style="word-break:break-all">Qi, X. J.; Lu, C. H.; Liu, X.; Shimron, S.; Yang, H. H.; Willner, I., Autonomous control of interfacial electron transfer and the activation of DNA machines by an oscillatory pH system. Nano letters 2013, 13 (10).</li>
<li style="word-break:break-all">Liang, X.; Kuhn, H.; Frank-Kamenetskii, M. D., Monitoring single-stranded DNA secondary structure formation by determining the topological state of DNA catenanes. Biophys J 2006, 90 (8), 2877-89.</li>
<li style="word-break:break-all">Jayaram M. Mathematical validation of a biological model for unlinking replication catenanes by recombination[J]. Proceedings of the National Academy of Sciences, 2013, 110(52): 20854-20855.</li>
<li style="word-break:break-all">Wang, A.H.-J., Quigley G.J., Kolpak, F.J., Crawford , J.L., van Boom, J.H., van der Marel, G., and Rich ,A.(1979).Molecular structure of a left-handed double helical DNA fragment at atomic resolution. Nature, 282,680-686.</li>
<li style="word-break:break-all">Hays, F.A., Teegarden, A., Jones, Z.J.R., Harms, M., Raup, D., Watson,. J., Caveliere, E., and Ho, P.S.(2005) How sequence defines structure: a crystallographic map of DNA structure and conformation. Proc. Natl. Acad Sci. USA, 102,7157-7162.
</li>
</ol>

</body>
</html>
    </div>

    

    

    <!-- Comments -->
    
    <div class="comments">
        


    </div>
    

</div>
        </section>
    </div>
</div>

</div>

<!-- Footer -->
<div class="push"></div>

<footer class="footer-content">
    <div class="container">
        <div class="row">
            <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 footer-about">
                <h2>Contact Us</h2>
                <p>
            This project was developed by Season 2016.If you have any questions,feel free to email 
			<a href="mailto:OUCBIOMOD@163.com">OUCBIOMOD@163.com</a>.
                </p>
            </div>
            
    <div class="col-xs-6 col-sm-6 col-md-3 col-lg-3 recent-posts">
        <h2>Laboratory</h2>
	<a href="http://www.drnatech.com/"><img src="lablogo.png"  height="80" width="465"></a>

    </div>



            
        </div>
        <div class="row">
            <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                <ul class="list-inline footer-social-icons">
                    
                    
                    <li class="list-inline-item">
                        <a href="https://twitter.com/?lang=en">
                            <span class="footer-icon-container">
                                <i class="fa fa-twitter"></i>
                            </span>
                        </a>
                    </li>
                    
                    
                    <li class="list-inline-item">
                        <a href="https://www.facebook.com/">
                            <span class="footer-icon-container">
                                <i class="fa fa-facebook"></i>
                            </span>
                        </a>
                    </li>
                    
                    

                                     

                    
                    <li class="list-inline-item">
                        <a href="https://plus.google.com/">
                            <span class="footer-icon-container">
                                <i class="fa fa-google-plus"></i>
                            </span>
                        </a>
                    </li>
                    
                    
                    <li class="list-inline-item">
                        <a href="http://mp.weixin.qq.com/s?__biz=MzIyNTUwNDgxMw==&mid=2247483686&idx=1&sn=48237c28853b1923a1bfcfe18f18a1f8&scene=23&srcid=0806nRkMAQ11mKdqxt3s5YKG#rd/">
                            <span class="footer-icon-container">
                                <i class="fa fa-wechat"></i>
                            </span>
                        </a>
                    </li>
                    
                    
                    <li class="list-inline-item">
                        <a href="mailto:OUCBIOMOD@163.com">
                            <span class="footer-icon-container">
                                <i class="fa fa-envelope-o"></i>
                            </span>
                        </a>
                    </li>
                    
                    
                    <li class="list-inline-item">
                        <a href="http://weibo.com/">
                            <span class="footer-icon-container">
                                <i class="fa fa-weibo"></i>
                            </span>
                        </a>
                    </li>
                    
		    
    

                </ul>
            </div>
        </div>
        <div class="row">
            <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                <div class="footer-copyright">
                    Copyright &copy; 2016 Team Season. All Rights Reserved.
                </div>
            </div>
        </div>
    </div>
</footer>

<!-- After footer scripts -->

<!-- jQuery -->
<script src="//code.jquery.com/jquery-2.1.4.min.js"></script>

<!-- Tween Max -->
<script src="//cdnjs.cloudflare.com/ajax/libs/gsap/1.18.5/TweenMax.min.js"></script>

<!-- Gallery -->
<script src="//cdnjs.cloudflare.com/ajax/libs/featherlight/1.3.5/featherlight.min.js" type="text/javascript" charset="utf-8"></script>

<!-- Custom JavaScript -->
<script src="../js/main.js"></script>

<!-- Disqus Comments -->



</body>

</html>
